src/others/components/Map.tsx

```diff
@@ -1,9 +1,48 @@
 import { useState } from "react";
-import MapComponent from "react-map-gl";
+import MapComponent, { Layer, LayerProps, Source } from "react-map-gl";
 import maplibregl from "maplibre-gl";
 import "maplibre-gl/dist/maplibre-gl.css";
 
 import styles from "./Map.module.css";
+import { aidRequestsFixture } from "../fixtures/request.fixture";
+import { FeatureCollection, GeoJsonProperties, Geometry } from "geojson";
+
+const layerStyle: LayerProps = {
```

Comment:
*non blocking*

Should probably live in a separate file, especially when additional layers come in.